frameworks/digital-outcomes-and-specialists/manifests/edit_brief.yml

```diff
@@ -57,128 +57,102 @@
     - summary
 
 -
-  name: Shortlist criteria
+  name: Shortlist and evaluation process
   editable: True
   step: 2
   description: |
-    After you’ve published your requirements, suppliers will apply and tell you:
+    There are 3 stages to finding the supplier that best meets your needs.
 
-    - when they can start work
-    - if they have the essential skills and experience
-    - which nice-to-have skills and experience they have
+    **1\. View suppliers who applied**<br />
```

Comment:
@allait says we can probably live with it as long as @superroz and @suedavis68 don't mind.  It's ugly stuff though.